osu.Game/Overlays/Chat/ChatLine.cs

```diff
@@ -291,6 +292,20 @@ public MenuItem[] ContextMenuItems
                     return items.ToArray();
                 }
             }
+
+            protected override bool OnHover(HoverEvent e)
+            {
+                this.FadeColour(new Color4(1.4f, 1.4f, 1.4f, 1), 150, Easing.OutQuint);
```

Comment:
While I understand this is technically correct, I don't know if it's something that's going to be supported forevermore.

Instead of this, I recommend using `.Lighten()` on the colour itself and assigning the result of that to the objects.

Reply:
Just pushed an implementation using `.Lighten()`. This was my original method for achieving this effect, but IMO it's... a bit clunky?